src/test/eo/org/eolang/collections/hash-code-of-tests.eo

```diff
@@ -42,30 +39,132 @@
           hash-code-of FALSE
     $.equal-to TRUE
 
-[] > hash-of-int-is-int
+[] > hash-code-of-int-is-int
   assert-that > @
     is-int.
       number
         hash-code-of 42
     $.equal-to TRUE
 
-[] > hash-codes-of-the-same-numbers-are-equal
+[] > hash-codes-of-the-same-int-are-equal-1
```

Comment:
@c71n93 I think it would be better to rename the test with `hash-codes-of-the-same-big-ints-are-equal`